core/domain/collection_services.py

```diff
@@ -1114,8 +1120,7 @@ def get_next_page_of_all_commits(
             commits starts from this datastore cursor. Otherwise, the returned
             commits start from the beginning of the full list of commits.
     Returns:
-        The return value is a triple (results, cursor, more) as described in
-        fetch_page() at:
+        3-tuple of (results, cursor, more) as described in fetch_page() at:
```

Comment:
Er, no -- I meant explain what 'results', 'cursor', 'more' are, in addition to linking to the reference.